Description: Compare the jishoop pointer with ENDPTR using the intended pointer type.
 Source commits: 30049dbe4d9d4b897c531d12d98e2e4ece44adbc
Origin: upstream
Forwarded: yes
Last-Update: 2026-07-14

diff --git a/Wnn/jserver/jishoop.c b/Wnn/jserver/jishoop.c
index 2fd9304..9252705 100644
--- a/Wnn/jserver/jishoop.c
+++ b/Wnn/jserver/jishoop.c
@@ -531,7 +531,7 @@ found_it:
   if (p->kosuu == 1)
     {
       *next_pter = p->next;
-      if (tary[ind1].pter == NULL)
+      if (tary[ind1].pter == ENDPTR)
         remove_space (tary, ind1, jtl, tary[ind1].pter1);
       if (p->serial == jtl->maxserial - 1)
         {
